src/Area.java

```diff
@@ -0,0 +1,158 @@
+
+Design a class Area to calculate areas for different shapes. It should support following methods
+Method to return area of a Triangle. Method will receive width and height as double and return computed value as double.
+Method to return area of a Rectangle. Method will receive width and height as double and return computed value as double.
+Method to return area of a Square. Method will receive width as double and return computed value as double.
+Method to return area of a Circle. Method will receive radius as double and return computed value as double.
+ 
+Assuming that height, width and radius will be greater than zero write proper specification for each method created above.
+Also assume that these methods may throw ArithmeticException.
+
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+
+public class Area
+{
+	static Scanner scanner = new Scanner(System.in);
+	private static String choices[] = new String[]{"Triangle","Rectrangle","Square","Circle","Exit"};
+	private static int choice=0;
+	private static double result;
+	public static void main(String args[])
+	{
+		new Area();
+	}
+	Area()
+	{
+		/*
+		 * Prints the available options through helper function
+		 * Invokes the operation functions as per choice shown
+		 */
+		printMenu();
+		navigate();
+	}
+	
+	private static void printMenu()
+	{
+		/*
+		 * Prints the operations available for user to perform
+		 */
+		String message="Calculate area of ";
+		System.out.println(message);
+		for(String value:choices)
+		{
+			System.out.println(++choice +" - "+value);
+		}
+		choice=getChoice();
+	}
+	private static void navigate()
+	{
+		/*
+		 * Takes input from golbal variable choice and invokes the function of choice
+		 * Outputs the area of choice in decimal form with percision of 2 decimal points
+		 */
+		switch(choice)
+		{
+			case 1: result=areaOfTriangle(getInput("Base"),getInput("Height"));break;
+			case 2: result=areaOfRectangle(getInput("Width"),getInput("Height"));break;
+			case 3: result=areaOfSquare(getInput("Width"));break;
+			case 4: result=areaoOfCircle(getInput("Radius"));break;
+			case 5: System.exit(-1);break;
+			default: System.out.println("Un-expected error");
+		}
+		System.out.printf("%s%.2f","Area of "+choices[choice-1]+" = ",result);
+	}
+	private static double getInput(String str)
+	{
+		/*
+		 * Takes the input from the user
+		 * Returns double value
+		 * Discard other intergral and string values
+		 */
+		double input=0;
+		System.out.println("Enter the  : "+str+" of "+choices[choice-1]);
+		try
+		{
+			input=scanner.nextDouble();
+			if(input>0)
+			{
+				return input;
+			}
+			else
+			{
+				input=getInput(str);
+			}
+		}
+		catch(InputMismatchException e)
+		{
+			System.out.println("Please enter Double value");
+			scanner.nextLine();
+			input=getInput(str);
```

Comment:
Same here, You should add a message for user here also.

Reply:
Resolved.